app/validators/claim/litigator_common_validations.rb

```diff
@@ -35,7 +35,7 @@ def validate_offence
     def validate_case_concluded_at
       validate_presence(:case_concluded_at, 'blank')
       validate_not_before(Settings.earliest_permitted_date, :case_concluded_at, 'check_not_too_far_in_past')
```

Comment:
suggestion - for consistent naming this could be `validate_on_or_after` "positive naming" if relatively easy to replace